### PR TITLE
[RW-6200][risk=no] adding PM to concept set search

### PR DESCRIPTION
**Pull request by freemabd**
adding PM to concept set search and dataset preview and export to notebook. This PR also bumps the synth/preprod/prod cloud CDR indices to new version that includes the new PM criteria rows.

Screen Shot 2021-02-17 at 10 27 46 AM

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java [hunk 1]
```diff
@@ -503,17 +468,20 @@ public Long findSurveyCount(String name, String term) {
...
+                    || (!s.startsWith("+") && !s.startsWith("-") && !s.endsWith(")"))) {
+                  return "+" + s;
+                }
+                return s;
+              })
+          .collect(Collectors.joining(" "));
     }
 
     String[] keywords = term.split("\\W+");
-    if (keywords.length == 1 && keywords[0].length() <= 3) {
```
Comment (NehaBroad) [inline api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java:514]: Why was this removed?

Comment (freemabd) [reply api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java:514]: This was mainly to handle exact matches on code, but the UI no longer allows words under 3 characters to be submitted and the dao checks codes with a like in the dao: https://github.com/all-of-us/workbench/blob/3f6a918b1ddc56fa32f0748d05159ad3e2858030/api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java#L159-L159

**Review by NehaBroad — APPROVED** [review]

**Review by dolbeew — APPROVED** [review]
Deployed locally, PM seems to be working in concept search and dataset builder
